scripts/helpers.py

```diff
@@ -666,15 +666,8 @@ def aggregate_fuels(sector):
     biomass_fuels = [
         "Bagasse",
         "Fuelwood",
-        "Peat",
-        "Peat products",
-        "Charcoal",
-        "Petroleum Coke",
-        "Lignite",
-        "Brown coal",
-        "Hrad coal",
-        "Sub-bituminous coal",
-        "Petroleum Coke",
+        # "Peat",
```

Comment:
These are now included somewhere else?

Reply:
Except peat and peat products, yes. Not sure how to handle these best, any idea @hazemakhalek?

Reply:
Why not add them in coal fuels? I understand you removed them because they have been misplaced to Biomass, right? @energyLS

Reply:
```suggestion
```